tools/src/main/python/opengrok_tools/mirror.py

```diff
@@ -107,12 +108,18 @@ def main():
                              ' repositories,'
                              ' terminate the processing'
                              ' if no change is found.')
+    parser.add_argument('--strip-outgoing', action='store_true', default=False,
```

Comment:
Maybe add a note that it's only supported by some repositories (e.g. git)? It seems to me that otherwise it's just ignored.

Reply:
Makes sense, added.